bpf/init.sh

```diff
@@ -257,10 +258,12 @@ function bpf_load()
 	OPTS="${OPTS} -DNODE_MAC=${NODE_MAC} -DCALLS_MAP=${CALLS_MAP}"
 	bpf_compile $IN $OUT obj "$OPTS"
 	tc qdisc replace dev $DEV clsact || true
-	[ -z "$(tc filter show dev $DEV $WHERE | grep -v 'pref 1 bpf chain 0 $\|pref 1 bpf chain 0 handle 0x1')" ] || tc filter del dev $DEV $WHERE
+	[ -z "$(tc filter show dev $DEV $WHERE \
+    | grep -v "pref $TC_FILTER_PRIO bpf chain 0 $\|pref $TC_FILTER_PRIO bpf chain 0 handle 0x1")" ] \
+      || tc filter del dev $DEV $WHERE
```

Comment:
Please indent the lines, to mark that they're the continuation of the one above.

Reply:
ah, I see. tabs vs. spaces. I will fix.

Reply:
imho, it reads better in a single line as it was

Reply:
I can do that. I've just been using 80 columns as a limit and wrapping. If that is not the cilium style, that's fine with me. ;) I have changed it back to a single line. Thanks.